src/fhir_mcp_server/server.py

```
@@ -271,6 +287,20 @@ async def search(
                 ],
             ),
         ],
+        fields: Annotated[
+            List[str],
+            Field(
+                description="FHIRPath expressions specifying which fields to return. Omitting returns the full resource.",
+                examples=[["Condition.code", "Condition.clinicalStatus","Bundle.total"]],
+            ),
+        ] = [],
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)